tests/integration_tests.rs

```diff
@@ -873,6 +873,42 @@ fn disable_pager_if_disable_paging_flag_comes_after_paging() {
         .stdout(predicate::eq("hello world\n").normalize());
 }
 
+#[test]
+fn disable_pager_if_pp_flag_comes_after_paging() {
+    bat()
+        .env("PAGER", "echo pager-output")
+        .arg("--paging=always")
+        .arg("-pp")
+        .arg("test.txt")
+        .assert()
+        .success()
+        .stdout(predicate::eq("hello world\n").normalize());
+}
+
+#[test]
+fn enable_pager_if_disable_paging_flag_comes_before_paging() {
+    bat()
+        .env("PAGER", "echo pager-output")
```

Comment:
Thank you for the rebase @J-Kappes .

@boyvanduuren I don't understand how this new test can pass on Windows. It is rebased on your work, so shouldn't this need `mocked_pagers::from()`? Is CI not working as we expect?

Reply:
> Is CI not working as we expect?

Seems so indeed. Running the tests locally on Windows/PS cause them to fail. I'll create a bug for this and take another look.